File: fizzbuzz/fizzbuzz.php
// Fizzbuzz

<?php
  for ($x = 1; $x <= 100; $x++) {
    if ($x % 3 == 0 && $x % 5 == 0) {
      echo "FizzBuzz\n";
    } elseif ($x % 3 == 0) {
      echo "Fizz\n";
    } elseif ($x % 5 == 0) {
      echo "Buzz\n";
    } else {
      echo "$x\n";
    }
  }
?>

File: fizzbuzz/fizzbuzz.rs
// fizzbuzz

fn main() {
  for x in 1..101 {
    let out =
      if x%3 == 0 && x%5 == 0 {
        "FizzBuzz"
      } else if x%3 == 0 {
        "Fizz"
      } else if x%5 == 0 {
        "Buzz"
      } else {
        "num" // x.to_string();
      };

    if out == "num" {
      println!("{}", x);
    } else {
      println!("{}", out);
    }
  }
}
